doc/architecture/receipts.rst

```diff
@@ -0,0 +1,9 @@
+Receipts
+========
+
+CCF implements `write` receipts, which are signed proofs associated with a transaction. They serve two main purposes:
```

Comment:
```suggestion
CCF implements `write` receipts, which are signed proofs associated with a transaction, i.e. a unique TxID. They serve two main purposes:
```

Reply:
I'm not sure this is clearer, the receipt is really bound to the whole transaction, not just the TxID. A transaction has a TxID, but that's not particularly central here?